### PR TITLE
fix: client-project-setup package

### PR DESCRIPTION
**Pull request by davelanglois-ssc**
- adding specific root-sync for tier3 and tier4
- Grant GCP Project IAM Admin to GCP SA on Client Host Project
- Grant GCP role Compute Security Admin to tier3-sa GCP SA
- enable logging and monitoring API
- add securitycontrols.md

### CODE REVIEW

**Comment by fmichaelobrien** [comment]
Thanks Dave, this one I will review a bit more - eta later tonight

**Review by fmichaelobrien — APPROVED** [review]
Lgtm
I have the core lz only on one of my orgs will add/retest this one to check the changes

Thanks Dave and team - all 4 PRs can be sequenced now

**Review by cartyc — DISMISSED** [review]
lgtm

File: solutions/client-project-setup/project.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ metadata:
   namespace: client-name-projects # kpt-set: ${client-name}-projects
   annotations:
     cnrm.cloud.google.com/auto-create-network: "false"
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/client-name-hierarchy/Folder/project-parent-folder # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/${client-name}-hierarchy/Folder/${project-parent-folder}
```
Comment (alaincormier-ssc) [inline solutions/client-project-setup/project.yaml:23]: this depends-on was removed in PR #435

File: solutions/client-project-setup/namespaces/project-id-tier3.yaml [hunk 2]
```diff
@@ -98,6 +98,24 @@ spec:
   role: organizations/org-id/roles/tier3.dnsrecord.admin # kpt-set: organizations/${org-id}/roles/tier3.dnsrecord.admin
   member: "serviceAccount:tier3-sa@project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:tier3-sa@${project-id}.iam.gserviceaccount.com
 ---
+# Grant GCP Project IAM Admin to GCP SA on Client Host Project
```
Comment (alaincormier-ssc) [inline solutions/client-project-setup/namespaces/project-id-tier3.yaml:101]: to be validated

File: solutions/client-project-setup/project.yaml (hunk 1 above)
Comment (davelanglois-ssc) [reply solutions/client-project-setup/project.yaml:23]: thank you very much for tracking this.

**Review by fmichaelobrien — APPROVED** [review]
lgtm

File: solutions/client-project-setup/project.yaml (hunk 1 above)
Comment (davelanglois-ssc) [reply solutions/client-project-setup/project.yaml]: removed in last commit

File: solutions/client-project-setup/namespaces/project-id-tier3.yaml (hunk 2 above)
Comment (davelanglois-ssc) [reply solutions/client-project-setup/namespaces/project-id-tier3.yaml]: removed in last commit as I have validated that it was an old requirement that no longer exist

**Review by alaincormier-ssc — APPROVED** [review]

**Pull request merged**